[1.18.2] Add Traditional Chinese (zh_tw) to Patchouli Book

Jimmy-sheep opened the pull request:
If there's any problem of translation, please let me know.
*Some of tooltips is basic on [MC 百科](https://www.mcmod.cn/item/list/3824-1.html).

Commit:
Add files (not yet start)

copy from en_us

Commit:
Translate "Categories" files

Commit:
Merge remote-tracking branch 'origin/1.18.2' into 1.18.2

Commit:
Replace CR(\r) to LF(\n)

Commit:
Replace CR(\r) to LF(\n)

Commit:
Replace CR(\r) to LF(\n)

Jimmy-sheep committed:
Delete 新文字文件.txt

Commit:
Re-translate hazmat suit

Replace `防護服` -> `防護套裝`

Commit:
Fix translate of Dynamo

Commit:
Translate "entries/equipment/armor" files

我盡力翻譯這句了`要$(o)飛$()常小心`
在beekeeper.json中原文為`$(o)Bee$() careful`

Commit:
Translate "entries/equipment/charges" files

Commit:
Re-translate "PNT"

`PNT` -> `植育催熟炸藥`

Commit:
Translate "entries/equipment/explosives" files

Commit:
Fix some translation

Commit:
Revert the translation

Commit:
Re-translate potion_duration_augment

Commit:
Re-translate the word "mine/mining"

礦物是用`挖`的不是用`採`的

Commit:
Translate "entries/equipment/tools" files

Commit:
Translate "RF" into "紅石通量"

Commit:
Re-translate "device"

`設備`->`裝置`

Commit:
Fix translate of "Catalytic Reclamation Chamber"

Commit:
Translate "entries/guides"(Part 1)

Commit:
Fix the translation

`通過` -> `藉由`

Commit:
Mini translate before this part start

Commit:
Translate "entries/guides"(Part 2)

Commit:
Translate "entries/guides"(Part 2.5, fix)

Commit:
Translate "entries/links/contributions"

對於 FloofHips 的最後一句`使這本書多次變得更加爵士樂。`我不確定翻譯是否正確

Commit:
Translate "entries/links"

Commit:
Translate "entries/storage_and_transport/locomotion"

Commit:
Re-translate "Flux Linkage Amplifier"

`通量連接放大器` -> `通量鏈結放大器`

Commit:
Fix some translate

`列表` -> `清單` 
And other translation chnages

Commit:
Re-translate the word "list"

`列表` -> `清單`

Commit:
Re-translate the word "slot"

`槽` -> `欄`

Commit:
Fix translate of "Tinker's Workbench"

Commit:
Translate "entries\technology\augments"

Commit:
Re-translate "Vacuumulator"

Commit:
Re-translate "Radial Enhancement" and "Axial Enhancement"

Commit:
Fix some translations

Commit:
Re-translate the word "作用"

Commit:
Re-translate "Igneous Extruder"

Commit:
Fix the translation

Commit:
Re-translate "Phyto-Soil"

Commit:
Translate "entries/technology/devices"

Commit:
Re-translate "Multi-cycle Injectors"

Commit:
Re-translate "Multi-cycle Injectors"

Commit:
Translate "entries/technology/dynamos"

Commit:
Re-translate "Sequential Fabricator"

Commit:
Re-translate "Pyrolyzer"

Commit:
Re-translate the word "stores"

Commit:
Revert the translate of "Redstone Furnace"

The translation is based on the input energy(Redstone Flux) not the name.

Commit:
Re-translate "Induction Smelter"

Commit:
Translate "entries/technology/machines" (Part 1)

Commit:
Re-translate the word "設置" and "設定"

`設置` use as a noun, `設定` use as a verb.

Commit:
Translate "entries/technology/machines" (Part 2)

Commit:
Translate other files in "entries/technology"

Commit:
Translate the file "catalyst_page"

Commit:
Fix the translation

`您` -> `你`

Commit:
Fix some translations

`設備` -> `裝置`
`儲存方塊` -> `貯存塊`

Commit:
Re-translate (Rename) the mob "Blitz"

`狂風使者` -> `雷霆使者`

Commit:
Translate "entries/world" (Part 1)

Commit:
Re-translate "Fluxtooth Mushroom"

Commit:
Re-translate "Latex"

Commit:
Translate "entries/world" (Part 2)

Commit:
Fix some translations...again

`發電`->`產能`
`發電機`->`能源爐`
`開採`->`挖掘`
`形式`->`格式`
`通過`->`藉由`
`用作`->`當作`

Commit:
Fix some translations...again and again

`發電機`->`能源爐`

Commit:
Fix some translation

Re-translate some tooltips.
Fix some format problem.

Commit:
Fix some translations

Re-translate "cheese"
Re-translate some tooltips.

Commit:
Fix quotation marks

Commit:
Re-translate "duct "

Commit:
Fix about "fluid_duct_windowed"

Jimmy-sheep committed:
Merge branch '1.18.2' into 1.18.2

StarskyXIII commented on src/main/resources/data/thermal/patchouli_books/guidebook/zh_tw/entries/guides/faq.json:
```diff
@@ -0,0 +1,35 @@
+    },
+    {
+      "type": "patchouli:text",
+      "text": "Q: 如何修改礦物生成？$(br)A: 該設置有啟用/停用礦物生成的選項，但尚未實現對生成高度等細節的調整。$(br2)Q: 我如何更改設置？$(br)A: 設置現在是在每次保存的基礎上建立；在儲存檔案中的 serverconfigs 資料夾中找到它們。你還可以藉由將檔案從 serverconfigs 資料夾複製到 defaultconfigs 資料夾並進行編輯來設定預設設置。"
```
儲存

StarskyXIII commented on src/main/resources/data/thermal/patchouli_books/guidebook/zh_tw/entries/technology/machines/introduction.json:
```diff
@@ -0,0 +1,36 @@
+    },
+    {
+      "type": "patchouli:text",
+      "text": "在「設置」面板中，方塊的每一側都由一個正方形表示，點擊時將循環顯示設置。藍色和橙色分別代表輸入和輸出。箭頭按鈕設置自動轉移到/從相鄰物品欄。黑色表示該側將接受輸入/輸出，但不會自動傳輸。拾取方塊時會保存側面設置。"
```
儲存

StarskyXIII commented on src/main/resources/data/thermal/patchouli_books/guidebook/zh_tw/entries/technology/redprint.json:
```diff
@@ -0,0 +1,17 @@
+  "pages": [
+    {
+      "type": "patchouli:text",
+      "text": "紅圖可以輕鬆複製和貼上你的方塊設置！要保存設置，請手持空白紅圖並右鍵點擊已設定的方塊。手持已設定紅圖並右鍵點擊其他的機器以貼上設置。Shift + 右鍵點擊可以清除紅圖。將紅圖放在副手欄中以自動設定放置的方塊。"
```
保存 -> 儲存
副手 -> 非慣用手

StarskyXIII commented on src/main/resources/data/thermal/patchouli_books/guidebook/zh_tw/categories/storage_and_transport.json:
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "儲存和運輸",
+  "description": "本節涉及存儲和移動諸如資源和生物之類的東西。",
```
存儲 -> 儲存

StarskyXIII commented on src/main/resources/data/thermal/patchouli_books/guidebook/zh_tw/entries/storage_and_transport/dynamics.json:
```diff
@@ -0,0 +1,39 @@
+  "pages": [
+    {
+      "type": "patchouli:text",
+      "text": "動力目前被擱置，因為熱力系列的其他功能正在充實。請耐心等待！以下頁面將有其他推薦的管道模組。"
```
```suggestion
      "text": "《熱動力學》目前暫被擱置，因為熱力系列的其他功能還在完善當中。請耐心等待！以下頁面將有其他推薦的管道模組。"
```

StarskyXIII commented on src/main/resources/data/thermal/patchouli_books/guidebook/zh_tw/entries/storage_and_transport/dynamics.json:
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "動力在哪裡？",
```
```suggestion
  "name": "《熱動力學》在哪裡？",
```

StarskyXIII commented on src/main/resources/data/thermal/patchouli_books/guidebook/zh_tw/entries/storage_and_transport/dynamics.json:
```diff
@@ -0,0 +1,39 @@
+      "title": "XNet",
+      "url": "https://www.curseforge.com/minecraft/mc-mods/xnet",
+      "link_text": "Curseforge 連結",
+      "text": "XNet是一款功能強大的模組，可以在一根纜線中傳輸物品、流體、紅石通量等。每個網絡都需要一個控制器，藉由它可以設定網絡以控制多個通道上的輸入、輸出、過濾、堆疊大小等。涉及到輕微的學習曲線，但控製程度幾乎是無與倫比的。"
```
```suggestion
      "text": "XNet 是一款功能強大的模組，可以在一條線纜中傳輸物品、流體、紅石通量等。每個網路都需要一台控制器，藉由它設定網路以控制多個頻道上的輸入、輸出、篩選、堆疊大小等。涉及到輕微的學習曲線，但可操作性幾乎是無與倫比的。"
```

Jimmy-sheep replied:
你好 關於`網絡`與`網路`的部分我有疑問 
在我認知中`網路`比較接近"電腦間相互通訊的網際網路"
而`網絡`比較接近"事物之間的關係"  又或是網格脈絡
因此這邊我選用的是`網絡`而非`網路` 如此你覺得應該翻譯為`網路`嗎?

StarskyXIII commented on src/main/resources/data/thermal/patchouli_books/guidebook/zh_tw/entries/storage_and_transport/dynamics.json:
```diff
@@ -0,0 +1,39 @@
+      "title": "Refined Pipes",
+      "url": "https://www.curseforge.com/minecraft/mc-mods/refined-pipes",
+      "link_text": "Curseforge 連結",
+      "text": "要獲得更像 1.12 動力的體驗，請嘗試 Refined Pipes。管道和提取器以不同的速度進入不同的層次。對於那些已經熟悉 1.12 動力的人來說，這個模組應該感覺像家一樣。"
```
```suggestion
      "text": "要獲得更像 1.12 熱動力學的體驗，請嘗試 Refined Pipes。管道和提取器以不同的速度進入不同的層次。對於那些已經熟悉 1.12 熱動力學的人來說，這個模組應該感覺像家一樣。"
```

StarskyXIII commented on src/main/resources/data/thermal/patchouli_books/guidebook/zh_tw/entries/storage_and_transport/dynamics.json:
```diff
@@ -0,0 +1,39 @@
+      "title": "Integrated Tunnels",
+      "url": "https://www.curseforge.com/minecraft/mc-mods/integrated-tunnels",
+      "link_text": "Curseforge Link",
+      "text": "Integrated Dynamics 是一個非常強大的模組，可以評估關於生物、物品欄等的各種品質。Tunnels 是一個插件，允許網絡也從物品欄中導入/導出項目。兩者相結合，可實現極高的精確度、篩選性和控制性。當心，這個模組比大多數模組更難上手。"
```
```suggestion
      "text": "Integrated Dynamics 是一個非常強大的模組，可以檢測關於生物、容器等的各種數值。Tunnels 是它的擴充模組，允許網路從容器中導入/導出項目。兩者相結合，可實現極高的精確度、篩選性和控制性。當心，這個模組比大多數模組更難上手。"
```

StarskyXIII commented on src/main/resources/data/thermal/patchouli_books/guidebook/zh_tw/entries/technology/machines/crafter.json:
```diff
@@ -0,0 +1,24 @@
+  "pages": [
+    {
+      "type": "patchouli:text",
+      "text": "序列裝配機是一種自動合成物品的機器。一旦設定了製作格線中的配方，它將從其內部物品欄中取出物品和液體並製作它們。使用一桶特定液體並留下空桶的配方可以改用內部儲罐中的液體製作。"
```
```suggestion
      "text": "序列裝配機是一種可自動合成物品的機器。一旦設定好了合成表上的配方，它將從其內部物品欄中取出物品和液體並進行製作。將流體儲存在內部儲罐中，可以取代那些會需要流體桶的合成配方。"
```

Jimmy-sheep replied:
謝謝 這翻譯好流暢 果然是專業的 😮

StarskyXIII commented on src/main/resources/data/thermal/patchouli_books/guidebook/zh_tw/entries/guides/faq.json:
```diff
@@ -0,0 +1,35 @@
+  "pages": [
+    {
+      "type": "patchouli:text",
+      "text": "Q: 動力/管道在哪裡？$(br)A: 請詳閱$(l:thermal:storage_and_transport/dynamics)此處$(/l)。$(br2)Q: 升級套件在哪裡？$(br)A: 請詳閱$(l:thermal:technology/augments/integral_components)此處$(/l)。$(br2)Q: 我該如何提升我的機器的速度？$(br)A: 請詳閱$(l:thermal:technology/augments/integral_components)此處$(/l)和$(l:thermal:technology/augments/machine_speed)此處$(/l)。$(br2)Q: 這些增強元件可以疊加嗎？$(br)A: 請詳閱$(l:thermal:technology/augments/introducti ...
```
```suggestion
      "text": "Q: 熱動力學/管道在哪裡？$(br)A: 請詳閱$(l:thermal:storage_and_transport/dynamics)此處$(/l)。$(br2)Q: 升級套件在哪裡？$(br)A: 請詳閱$(l:thermal:technology/augments/integral_components)此處$(/l)。$(br2)Q: 我該如何提升我的機器的速度？$(br)A: 請詳閱$(l:thermal:technology/augments/integral_components)此處$(/l)和$(l:thermal:technology/augments/machine_speed)此處$(/l)。$(br2)Q: 這些增強元件可以疊加嗎？$(br)A: 請詳閱$(l:thermal:technology/augments/introduction)此處$(/l)。"
```

StarskyXIII commented on src/main/resources/data/thermal/patchouli_books/guidebook/zh_tw/entries/guides/faq.json:
```diff
@@ -0,0 +1,35 @@
+    },
+    {
+      "type": "patchouli:text",
+      "text": "Q: 我該在哪裡以及如何回報錯誤？$(br)A: 你可以在我們的$(l:thermal:links/discord)Discord 伺服器$(/l)中詢問。在回報之前，請確保你的模組是最新的並且你沒有安裝 Optifine。$(br2)Q: 我該如何改變配方？$(br)A: 熱力模組都是資料驅動的，因此你可以使用資料包修改配方，如$(l:thermal:guides/modpack/recipes)此處$(/l)所示。或者，熱力模組具有 CraftTweaker 支持。"
```
```suggestion
      "text": "Q: 我該在哪裡以及如何回報錯誤？$(br)A: 你可以在我們的$(l:thermal:links/discord)Discord 伺服器$(/l)中詢問。在回報之前，請確保你的模組是最新的並且你沒有安裝 Optifine。$(br2)Q: 我該如何改變配方？$(br)A: 熱力模組都是資料驅動的，因此你可以使用資料包修改配方，如$(l:thermal:guides/modpack/recipes)此處$(/l)所示。或者，熱力模組具有 CraftTweaker 支援。"
```

StarskyXIII commented on src/main/resources/data/thermal/patchouli_books/guidebook/zh_tw/entries/guides/modpack/recipes.json:
```diff
@@ -0,0 +1,38 @@
+      "type": "patchouli:link",
+      "url": "https://minecraft.fandom.com/wiki/Data_pack",
+      "link_text": "Minecraft Wiki",
+      "text": "所有機器、裝置、能源爐等都使用 JSON 來決定它們的配方和催化劑，這意味著你可以使用資料包來建立和修改配方。或者，熱力具有 CraftTweaker 支持，如果你對此更滿意的話。下方是 Minecraft Wiki 的連結，可以幫助你開始使用資料包。"
```
```suggestion
      "text": "所有機器、裝置、能源爐等都使用 JSON 來決定它們的配方和催化劑，這意味著你可以使用資料包來建立和修改配方。或者，熱力具有 CraftTweaker 支援，如果你對此更滿意的話。下方是 Minecraft Wiki 的連結，可以幫助你開始使用資料包。"
```

StarskyXIII commented:
`Components` 在台灣的主流翻譯應該會是 `元件`。
https://www.microsoft.com/zh-tw/language/Search?&searchTerm=Components&langID=129&Source=true&productid=0

所以 `整合組件` 改譯作 `整合元件` 可能會更合適。

`Fluid` -> `流體`
`Liquid` -> `液體`
因為我不太想 fork 專案，就為了用 VSCode 查哪些地方 fluid 誤翻成液體。
因此要辛苦你從你 fork 的儲存庫慢慢查了

至於 `增強相容性` 改譯作 `可安裝的升級` 或 `可安裝的增強元件` 可能更為直覺？

Jimmy-sheep commented:
> `Components` 在台灣的主流翻譯應該會是 `元件`。

奇怪 怎麼記得之前翻譯得到的是組件...
我會全面修改一次 謝謝

哦 我想到了 我是根據`硬化整合組件` 這幾個物品的名稱取用`組件`的 那我物品名稱也需要修改嗎?

Commit:
Fix some translations(Part 1)

Jimmy-sheep commented:
> `Fluid` -> `流體`
> `Liquid` -> `液體`

這部分我想提到一下 `Potion Fluid`這邊我翻譯為`藥水溶液` 我認為翻譯成`藥水流體`怪怪的 因此想徵詢遺下你的意見 如果你認為應該翻譯為`藥水流體` 那麼我會全部變更的

Commit:
Fix some translations(Part 2)

Fix `fluid` translate into `液體`
(`液體`->`流體`)
Fix the quotation mark

StarskyXIII commented:
> 

原文是 `Components` 在台灣的主流翻譯應該會是 元件。

> > `Fluid` -> `流體`
> > `Liquid` -> `液體`
> 
> 這部分我想提到一下 `Potion Fluid`這邊我翻譯為`藥水溶液` 我認為翻譯成`藥水流體`怪怪的 因此想徵詢遺下你的意見 如果你認為應該翻譯為`藥水流體` 那麼我會全部變更的

我覺得你這樣翻得很好啊

Commit:
Re-translate "Component"

Jimmy-sheep commented:
`元件`的部分我修正了 另外我想確認一下我上方提到的 `網路`或`網絡`的問題

> 你好 關於`網絡`與`網路`的部分我有疑問
> 在我認知中`網路`比較接近"電腦間相互通訊的網際網路"
> 而`網絡`比較接近"事物之間的關係" 又或是網格脈絡
> 因此這邊我選用的是`網絡`而非`網路` 如此你覺得應該翻譯為`網路`嗎?

Jimmy-sheep committed:
Merge branch 'CoFH:1.18.2' into 1.18.2

StarskyXIII commented:
> `元件`的部分我修正了 另外我想確認一下我上方提到的 `網路`或`網絡`的問題
> 
> > 你好 關於`網絡`與`網路`的部分我有疑問
> > 在我認知中`網路`比較接近"電腦間相互通訊的網際網路"
> > 而`網絡`比較接近"事物之間的關係" 又或是網格脈絡
> > 因此這邊我選用的是`網絡`而非`網路` 如此你覺得應該翻譯為`網路`嗎?

XNet 傳輸是由一台控制器（就像是電腦伺服器）所控制，與其他管道模組用扳手調整輸入輸出不太一樣。
模組中也有路由器、訊號天線等類似現實網路設備的功能性方塊。
關於 XNet 的物品可以參考我們自製的翻譯資源包：
https://github.com/xMikux/ModsTranslationPack/blob/main/assets/xnet/lang/zh_tw.json

Jimmy-sheep commented:
> XNet 傳輸是由一台控制器（就像是電腦伺服器）所控制，與其他管道模組用扳手調整輸入輸出不太一樣。 模組中也有路由器、訊號天線等類似現實網路設備的功能性方塊。 關於 XNet 的物品可以參考我們自製的翻譯資源包： https://github.com/xMikux/ModsTranslationPack/blob/main/assets/xnet/lang/zh_tw.json


XNet的部分我依據你所述的 改為`網路` 而Integrated Tunnels(集成管道)的部分 我看這模組似乎就是傳統的管線了
 你認為我也應該修改翻譯為`網路` 還是應該保持`網絡`的翻譯 亦或是翻譯為`管道`更能讓使用者明瞭?

* 抱歉這周電腦送去檢修 最近電腦狀況異常 在我更換新電腦前可能更新和回覆會較慢...

Commit:
Fix the translation of the word "network" in XNet

Commit:
Merge remote-tracking branch 'origin/1.18.2' into 1.18.2

Jimmy-sheep committed:
Merge branch 'CoFH:1.18.2' into 1.18.2

Commit:
Fix some translations

Commit:
Fix some translations

Commit:
Re-translate the string "Augment Compatibility"

`增強相容性`->`增強元件相容性`

Commit:
Revert the translation

Jimmy-sheep committed:
Merge branch 'CoFH:1.18.2' into 1.18.2

Commit:
Update zh_tw.json

`裝填`->`啟動` (檔案：遙控引爆器)
修正逗點格式
`丨`->`｜` (修改直槓從中文丨(ㄍㄨㄣˇ)改為標點符號)
修正Servo的翻譯 `伺服器`(server)->`伺服`(servo)
更新/修正部分翻譯

Jimmy-sheep committed:
Merge branch 'CoFH:1.18.2' into 1.18.2

Commit:
Fix the keyword

Commit:
Fix some translations

Re-compared to en_us file to match the original sentences.

Commit:
Merge remote-tracking branch 'origin/1.18.2' into 1.18.2

Jimmy-sheep commented:
@StarskyXIII 抱歉打擾了 我剛才再次對照了en_us語言檔案 並再次修改了部分翻譯 想確認一下目前的翻譯有沒有需要更改的地方 如果沒有的話 我大概下周假日就要把狀態改為Ready for review 並把檔案上傳到1.19.X分支中了 再次感恩你的檢查

Commit:
Re-name "Enderium"

對比地獄的"獄髓" 這個則是終界的"終焉"
根據patchouli手冊中ctm.json裡的小知識
"`終焉`因其扭曲時空的終界特性而得名！"
取因其扭曲時空而使萬物終結(終焉) 為其命名

Commit:
Re-translate mode "empty"

Commit:
Update zh_tw.json

Jimmy-sheep committed:
Merge branch 'CoFH:1.18.2' into 1.18.2

Commit:
Remove additional strings

Also fix some strings to match sentences in en_us file.

StarskyXIII commented:
好奇為何突然將 ``Enderium`` 重新命名為 ``終焉``。
Thermal 與多款模組都有聯動，你在官方這邊修改了譯名，
表示所有其他跟 Thermal 整合的模組，譯名都要跟著變動。

既然你決定變動了 ``Enderium``，那我想是時候來討論 ``Signalum`` 的譯名，
畢竟對岸 ``Signal`` 的翻譯叫作 ``信號``，但在台灣這邊是譯作 ``訊號``。
那我想應該不再適合繼續譯作 ``信素`` 了。

StarskyXIII commented:
關於 ``Enderium`` 自所以被譯作 ``終界合金``，是過去與舞秋風討論過的結果。
不過歡迎討論
![image](https://user-images.githubusercontent.com/71606873/218301451-a42963db-bccc-4401-b5f4-662547da5e75.png)

Jimmy-sheep commented:
> 關於 `Enderium` 自所以被譯作 `終界合金`，是過去與舞秋風討論過的結果。 不過歡迎討論 !

哦哦 原來是有如此討論過後的 抱歉我沒有考慮到與其他模組連動的影響 既然如此我還是將其改回`終界合金`好了

至於合金的部分其實我原本也有考慮到`獄髓` 亦也是遠古遺骸中的生鏽金屬片與黃金合成後的產物 雖說對岸是翻譯為`下界合金`但我想zh_tw中既然沒有也採用合金的名稱 那麼對應終界的金屬應當也不採用合金的名稱 而且其他的同系列金屬`信素`和`流明`也都是合金
不過這部分算是我的一己之私 為了避免各位翻譯前輩們的困擾我還是改回去好了

另外`信素`的部分我想也還是先保留原本的名稱 (直接改為`訊素`似乎也不太好 念起來像"迅速"...)

Commit:
Revert the name of "Enderium"

Jimmy-sheep commented:
關於`Enderium`這個金屬我想再補充一下：
事實上會想更改這個金屬的原因 大部分是因為我的強迫症作祟 在我看來熱力系列新增了大量的金屬合金 但除了`Enderium`都沒有`合金`的名稱 同時與此同系列的`Signalum(信素)`和`Lumium(流明)`金屬也都是以兩個字為名 偏偏`Enderium`是四個字 因此我在一開始就起了想更改這個合金名稱的念頭

考量到`Enderium`主要是以終界為背景 由於我不是國文系為主修 因此我詞彙量不多 我也只想出了3個名稱 分別是`終界``終末``終焉`
對比地獄出產的`獄髓`並沒有直接翻譯為`地獄錠` 因此我首先就把`終界錠`的名稱劃掉了
再者考慮到`終末錠` 然而我查找配方後並沒有發現`Enderium`與終末都市(遊戲中進度的名稱)有關 既沒有以界伏蚌為材料 也沒有與紫珀專系列相關 因此我也把`終末`給劃掉了
故而最終採用`終焉`的名稱 符合終界的背景 也能對應`獄隨`這個地獄產物 儘管並沒有完全能像`獄髓`參考到原材料(遠古遺骸<->髓)而起名 但我也想不到更好/流暢的名稱 引此採用這個

不過我並沒有堅持必須為`Enderium`改名 實在不希望造成人們的困擾 這裡我也只是想陳述一下我將其改名的由來而已 沒有別的意思 因此如我上方所說的還是先以`終界合金`為名

StarskyXIII commented:
> 至於合金的部分其實我原本也有考慮到`獄髓` 亦也是遠古遺骸中的生鏽金屬片與黃金合成後的產物 雖說對岸是翻譯為`下界合金`但我想zh_tw中既然沒有也採用合金的名稱 那麼對應終界的金屬應當也不採用合金的名稱 而且其他的同系列金屬`信素`和`流明`也都是合金

我覺得這個想法很好啊，畢竟新版 Thermal 系列模組，就是著重在延伸原版玩法，
那麼譯名想保留或是貼合原版風格，是能夠讓人理解的。

Jimmy-sheep commented:
避免跟其他模組的名稱混淆 還是先保持原樣 等以後有機會再變更好了...
其餘的翻譯如果有問題 也可以提出跟我說 我會修正的 感恩

Commit:
Add and fix some translations

Commit:
Fix some translations

Commit:
Fix some translations

Commit:
Fix the typo

Jimmy-sheep committed:
Merge branch 'CoFH:1.18.2' into 1.18.2

StarskyXIII commented on src/main/resources/assets/thermal/lang/zh_tw.json:
```diff
@@ -136,13 +136,13 @@
-  "info.thermal.secondary_mod": "副產物產生倍數",
+  "info.thermal.energy_mod": "能量倍率",
+  "info.thermal.primary_mod": "主產物倍率",
+  "info.thermal.secondary_mod": "副產物倍率",
```
我個人不認為這樣的改譯，能使玩家更清楚該資訊的意義

StarskyXIII commented:
還有我注意到 ``Stuffed Pepper`` 被翻作 ``甜椒塞牛肉``，
在模組本身配方中，肉類其實接受任何來自 forge:foods/meat/cooked 標籤的物品，
故譯作 ``甜椒盅`` 會更合適。

還有 ``Radish`` 的翻譯未被統一，有 ``小蘿蔔`` 以及我當初推薦的 ``櫻桃蘿蔔`` 翻譯

StarskyXIII commented:
順便測試了一下，這邊的 ``Transfer``，意指單位時間內，單元每面的能量傳輸上限。
但你 Cofh Core 那邊的 PR 已經被合併了，可能要麻煩您再 PR 修正。
![image](https://user-images.githubusercontent.com/71606873/219793300-94b3cceb-eb52-45a6-8cc5-55f2533b24a6.png)
![image](https://user-images.githubusercontent.com/71606873/219793384-d1198156-c521-48d9-b672-34bea9ad4dd7.png)

Jimmy-sheep committed:
Fix some translations

Re-translate "Stuffed Pepper"
Fix "Radish"

Jimmy-sheep committed:
Merge remote-tracking branch 'origin/1.18.2' into 1.18.2

Jimmy-sheep committed:
Re-translate the keyword

`安全`與`安全性`在搜尋上相似 故改為也許會用到的`私有`

Jimmy-sheep committed:
Merge branch 'CoFH:1.18.2' into 1.18.2

Jimmy-sheep committed:
Merge branch 'CoFH:1.18.2' into 1.18.2
